test/test-google-sheets-api.ts

```diff
@@ -0,0 +1,87 @@
+import test from "ava";
+import * as fs from "fs";
+import * as path from "path";
+import * as dot from "dotenv";
+import * as c from "chalk";
+import { google } from "googleapis";
```

Comment:
How does this get imported? Am I not seeing something in the `package.json`?

Reply:
It was in the `package.json` so we don't see it added in this PR! https://github.com/eddieantonio/predictive-text-studio/blob/production/package.json#L68 ([related PR](https://github.com/eddieantonio/predictive-text-studio/commit/d8362095e40c71c08b25af13a3e4978bd3ff5d10#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519))

Reply:
oh derp, thanks for pointing that out!